### PR TITLE
AzureStack AzureConsistentStorage update for branch Release 1.2.4

### PR DESCRIPTION
**Pull request by eridai**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@eridai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (eridai). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/922/

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/packages.config [hunk 1]
```diff
@@ -13,11 +14,9 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.26.305102204" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.0" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime" version="1.8.2" targetFramework="net45" />
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/packages.config]: if you remove this package from configuration you need to remove it from csproj as well. Then you will need to regenerate wxi file.

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Acquisition/DeleteAcquisition.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+using System.Globalization;
+using System.Management.Automation;
+using System.Net;
+using Microsoft.AzureStack.AzureConsistentStorage;
+
+namespace Microsoft.AzureStack.AzureConsistentStorage.Commands
+{
+    /// <summary>
+    /// 
+    /// </summary>
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Acquisition/DeleteAcquisition.cs]: no text in summary?

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Acquisition/GetAcquisition.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+using System.Management.Automation;
+using Microsoft.AzureStack.AzureConsistentStorage;
+using Microsoft.AzureStack.AzureConsistentStorage.Models;
+
+namespace Microsoft.AzureStack.AzureConsistentStorage.Commands
+{
+
+    /// <summary>
+    /// 
+    /// </summary>
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Acquisition/GetAcquisition.cs]: please add text

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Farm/AddFarm.cs [hunk 4]
```diff
@@ -51,9 +55,16 @@ public sealed class AddAdminFarm : AdminCmdlet
         [ValidateNotNull]
         public string Location{ get; set; }
 
+        /// <summary>
+        ///
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Farm/AddFarm.cs]: text

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Farm/ClearDeletedAccounts.cs [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        /// <summary>
+        /// farm identifier
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 4)]
+        [ValidateNotNull]
+        public string FarmName { get; set; }
+        
+        /// <summary>
+        /// 
+        /// </summary>
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Farm/ClearDeletedAccounts.cs]: text missing

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Share/AddShare.cs [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.AzureStack.AzureConsistentStorage;
+
+namespace Microsoft.AzureStack.AzureConsistentStorage.Commands
+{
+
+    /// <summary>
+    /// 
+    /// </summary>
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Share/AddShare.cs]: text missing

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/packages.config (hunk 1 above)
Comment (eridai) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/packages.config]: Thanks for the comments. They are fixed with a new commit.

**Pull request merged**